examples/PyCricket v1.0/cricketscore.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jun  4 15:59:29 2017
+
+@author: LALIT ARORA
+"""
+
+from pycricbuzz import Cricbuzz
+from twilio.rest import Client
+c = Cricbuzz() 
+matches = c.matches()
+message=""
+livematches=[]
+
+# printing live matches at that time
+for match in matches:
+    if(match['mchstate']=='inprogress'):
+        livematches.append(match['mchdesc'])
+print("Live Matches :")       
+for i in range(len(livematches)):
+    print(livematches[i])        
+      
+# Get your country match by this search 
+
+for match in matches:
+    if('IND ' in match['mchdesc']):
+        print(match['id'])
+
+#Get the feed for your team     
+print()   
+finalscorecard=""
+for match in matches:
+    if 'IND vs PAK' in match['mchdesc']:

```

Comment:
If this is getting the score for one team only, regardless of opponent, then this should be something like `if 'IND' in match['mchdesc']:`